lib/kafo/kafo_configure.rb

```diff
@@ -312,33 +312,33 @@ def app_option(*args, &block)
 
     def set_app_options
       app_option ['--[no-]colors'], :flag, 'Use color output on STDOUT',
-                            :default => !!config.app[:colors]
+                            :default => !!config.app[:colors], :advanced => true
       app_option ['--color-of-background'], 'COLOR', 'Your terminal background is :bright or :dark',
-                            :default => config.app[:color_of_background]
+                            :default => config.app[:color_of_background], :advanced => true
       app_option ['--dont-save-answers'], :flag, "Skip saving answers to '#{self.class.config.answer_file}'?",
-                            :default => !!config.app[:dont_save_answers]
+                            :default => !!config.app[:dont_save_answers], :advanced => true
       app_option '--ignore-undocumented', :flag, 'Ignore inconsistent parameter documentation',
-                            :default => false
+                            :default => false, :advanced => true
       app_option ['-i', '--interactive'], :flag, 'Run in interactive mode'
```

Comment:
I wonder how common and useful this option is. I haven't ever seen anybody who uses it, personally. Is that different in your experience @ekohl and would you be strongly opposed to marking this one as `advanced` as well ?

Reply:
I can't speak about how commonly used it is, but I think we shouldn't consider it advanced.

Reply:
I tagged @ekohl because I think I recall he has mentioned something about this option in the past, but I'm also interested to know your reasoning for passing this one over in the first place @ehelms

Reply:
I think interactive is a valid option, and there is nothing advanced about it's use. It is a method for installing with some guidance, and we do not sell it to our users broadly enough. But I do not think that warrants advanced case.